.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2.1
+
+orbs:
+  composer: itinerisltd/composer@0.2
+
+workflows:
+  test:
+    jobs:
+      - composer/install:
+          name: test
+          executor: composer/seven
+          pre-steps:
+            - run: sudo docker-php-ext-enable xdebug
+            - run: curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ~/cc-test-reporter
+            - run: chmod +x ~/cc-test-reporter
+            - run: ~/cc-test-reporter before-build
+          post-steps:
+            - composer/exec:
+                command: test
+            - run:
+                command: ~/cc-test-reporter after-build --coverage-input-type clover --exit-code $?
```

Comment:
I had to do

```
cc-test-reporter format-coverage -t clover coverage/clover.xml
cc-test-reporter upload-coverage
```

because after-build expects clover.xml in either root or some random dir instead of coverage/clover.xml